README.md

```diff
@@ -5,6 +5,54 @@
 Giant Swarm offers a vertical-pod-autoscaler Managed App which can be installed in tenant clusters.
 Here we define the vertical-pod-autoscaler chart with its templates and default configuration.
 
+Please note that this app is based on this [upstream chart](https://github.com/cowboysysop/charts/tree/master/charts/vertical-pod-autoscaler) so if some changes need to be applied to the vpa configuration, one may check the upstream values file to check out the additional fields that can be added to our own values.
+
+## Changes since migration to new upstream repo
```

Comment:
Maybe show versions number where this has changed, and add a link to here in the changelog?